src/build.xml

```diff
@@ -1,18 +1,18 @@
 <Project
     xmlns="http://schemas.microsoft.com/developer/msbuild/2003"
-    ToolsVersion="4.0"
+    ToolsVersion="15.0"
```

Comment:
does this enforce a minimum tools version or older versions of vs/msbuild will likely still be able to build this .sln?

Reply:
@mjkkirschner This is essentially a noop.. see this blurb regarding this setting on MSDN:

`Starting in Visual Studio 2013, the MSBuild Toolset version is the same as the Visual Studio version number. MSBuild defaults to this Toolset within Visual Studio and on the command line, regardless of the Toolset version specified in the project file. This behavior can be overridden by using the -ToolsVersion flag.`